### PR TITLE
Typo "Teams Room Pro"→"Teams Rooms Pro"

### PR DESCRIPTION
**Pull request by hyoshioka0128**
https://learn.microsoft.com/en-us/microsoftteams/proxy-servers-for-skype-for-business-online 
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/blob/live/Teams/proxy-servers-for-skype-for-business-online.md 
#PingMSFTDocs

### CODE REVIEW

File: Teams/proxy-servers-for-skype-for-business-online.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ Microsoft also strongly recommends:
     
 - Allowing UDP traffic for Teams Media (3478-3481) through your firewall
 
-- Allowing all required URLs and IPs, including those for Azure, Office 365, Intune and Teams Room Pro (where used) through your firewall
+- Allowing all required URLs and IPs, including those for Azure, Office 365, Intune and Teams Rooms Pro (where used) through your firewall
```
Comment (JohanFreelancer9) [inline Teams/proxy-servers-for-skype-for-business-online.md:69]:
```suggestion
- Allowing all required URLs and IPs, including those for Azure, Office 365, Intune, and Teams Rooms Pro (where used) through your firewall
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@hyoshioka0128, kindly commit the minor additional change, please. Many thanks.

**Comment by hyoshioka0128** [comment]
@JohanFreelancer9 - Thanks for the suggestion.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@MikePlumleyMSFT 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**